photutils/psf/groupstars.py

```diff
@@ -0,0 +1,141 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform source grouping."""
+
+from __future__ import division
+import abc
+import numpy as np
+from astropy.table import Column, Table, vstack
+
+
+__all__ = ['DAOGroup']
+
+
+class GroupStarsBase(object):
```

Comment:
you need to add it to the public classes to ``__all__`` to get rid of the sphinx warnings. (It is the base group for DAOGroup, thus the docs generation for the DAOGroup API tries to make a link to it, but since it's not public cannot find it).